string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,20 @@
+/* exported isVowel */
+
+function isVowel(char) {
+  if (
+    char === 'a' ||
+    char === 'e' ||
+    char === 'i' ||
+    char === 'o' ||
+    char === 'u' ||
+    char === 'A' ||
+    char === 'E' ||
+    char === 'I' ||
+    char === 'O' ||
+    char === 'U'
+  ) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
This would be a good use case for a `switch` statement.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```